packaging/PLATFORM_SUPPORT.md

```diff
@@ -0,0 +1,166 @@
+<!--
+title: "Netdata platform support policy"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/PLATFORM_SUPPORT.md
+-->
+
+# Netdata platform support policy
+
+Netdata defines three tiers of official support: Core, Intermediate, and Community. Each tier defines different
+guarantees for platforms in that tier, described below in the section about that tier.
+
+Additionally, we define two categories for special cases that we do not support, third-party supported platforms,
+and previously supported platforms. These two categories are explained further below.
+
+Any platforms not listed in any of these categories may or may not work
+
+A general outline of the various support tiers and categories is shown in the following table.
+
+|   | Bug Support | Guaranteed Configurations | CI Coverage | Native Packages | Static Build Support |
+| - | ----------- | ------------------------- | ----------- | --------------- | -------------------- |
+| Core | High priority | Everything but rare edge cases | Full | Yes, if we can provide them | Full |
+| Intermediate | Normal priority | Common cases | Partial (CI mostly equivalent to **Core**, but possibly with some gaps, and not required to pass) | Possibly | Full |
+| Community | Best Effort | Default only | None | No | Best Effort |
+| Third-party Supported | Users directed to platform maintainers | None | None | No | Best Effort |
+| Previously Supported | Users asked to upgrade | None | None | Yes, but only already published versions | Best Effort |
+
+- ‘Bug Support’: How we handle of platform-specific bugs.
+- ‘Guaranteed Configurations’: Which runtime configurations for the agent we try to guarantee will work with minimal effort from users.
+- ‘CI Coverage’: What level of coverage we provide for the platform in CI.
+- ‘Native Packages’: Whether we provide native packages for the system package manager for the platform.
+- ‘Static Build Support’: How well our static builds are expected to work on the platform.
+
+## Currently supported platforms
+
+### Core
+
+Platforms in the core support tier are our top priority. They are covered rigorously in our CI, usually
+include official binary packages, and any platform-specific bugs receive a high priority. From the perspective
+of our developers, platforms in the core support tier _must_ work, with almost no exceptions. Our [static
+builds](#static-builds) are expected to work on these platforms if available. Source-based installs are expected
+to work on these platforms with minimal user effort.
+
+| Platform | Version | Official Native Packages | Notes |
+| -------- | ------- | ------------------------ | ----- |
+| Alpine Linux | 3.15 | No | The latest release of Alpine Linux is guaranteed to remain at **Core** tier due to usage for our Docker images |
```

Comment:
@Ferroin Alpine Linux is listed as a supported platform. The documentation file about how to install Netdata on Alpine will be deleted in this pull request. 
Does the new kickstart script run on Alpine, so we don't need this info any more?

Reply:
Clarifying here despite discussing this on Slack so it’s visible to the community:

This is correct. We previously needed some extra steps to be done by the user prior to attempting to install on Alpine, but the new script does not require any of that.